### DISCUSSION TITLE
Downloading images and image galleries

### DISCUSSION THREAD

**Discussion by watcher1025** [aclap-dev/video-downloadhelper#1215] [Help and Q&A]
### Your question

Is this no longer supported ?

### Addon & CoApp version and details

Version 9.1.0.42 (stable), target mozilla, locale en-US.

Companion App installed (v2.0.19)

**Comment by jcfrog** [comment]
Sorry, it is no longer supported.

**Comment by Mattmjs** [comment]
There might be other extensions that do what you want to do.
